jetty-client/src/test/java/org/eclipse/jetty/client/http/HttpReceiverOverHTTPTest.java

```diff
@@ -262,4 +275,70 @@ protected void fillInterested()
         assertNotNull(response);
         assertEquals(200, response.getStatus());
     }
+
+    @ParameterizedTest
+    @MethodSource("complianceModes")
+    void testReceiveReentrance(HttpCompliance compliance) throws Exception
+    {
+        Log.getLogger(HttpReceiver.class).setDebugEnabled(true);
```

Comment:
This leaves the HttpReceiver in DEBUG state even after this testcase.
Can you remember the Level before it's used, and reset it to the remembered value after this testcase please?

Reply:
That's reasonable, let me do that.